sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/Microsoft.Azure.Services.AppAuthentication.csproj

```diff
@@ -2,7 +2,7 @@
     <PropertyGroup>
         <PackageId>Microsoft.Azure.Services.AppAuthentication</PackageId>
         <Description>Enables a service to authenticate to Azure services using the developer's Azure Active Directory/ Microsoft account during development, and authenticate as itself (using OAuth 2.0 Client Credentials flow) when deployed to Azure.</Description>
-        <Version>1.3.1</Version>
+        <Version>1.3.2</Version>
```

Comment:
I think you should bump minor. Packages roll forward on patch numbers

Reply:
done

Reply:
@AlexGhiondea could you take another look?